python/ml4ir/base/io/local_io.py

```diff
@@ -251,3 +252,39 @@ def rm_file(self, file_path: str):
         if os.path.isfile(file_path):
             os.remove(file_path)
             self.log("File deleted : {}".format(file_path))
+
+    def save_numpy_array(self, np_array, file_path: str, allow_pickle=True, zip=True, **kwargs):
```

Comment:
@jakemannix I had to do this because there is an issue with saving a list of numpy arrays directly and tensorflow layer weights are lists of arrays unfortunately. `savez` internally saves each list item in separate files and abstracts this out for us.

Reply:
Added.